app/components/ClientCredentialForm.tsx

```diff
@@ -0,0 +1,107 @@
+import type { DataFunctionArgs } from '@remix-run/node'
+import { useLoaderData } from '@remix-run/react'
+import { Label, TextInput, Dropdown, Button } from '@trussworks/react-uswds'
+import { useState } from 'react'
+import ReCAPTCHA from 'react-google-recaptcha'
+import { ClientCredentialVendingMachine } from '~/routes/user/client_credentials.server'
+import type { ClientCredentialData } from './ClientCredentialCard'
+
+export async function loader({ request }: DataFunctionArgs) {
+  const machine = await ClientCredentialVendingMachine.create(request)
+  const client_credentials = await machine.getClientCredentials()
+  return { client_credentials }
+}
+
+export default function ClientCredentialForm({
+  setItems,
+}: {
+  setItems: React.Dispatch<React.SetStateAction<ClientCredentialData[]>>
+}) {
+  const { client_credentials } =
+    useLoaderData<Awaited<ReturnType<typeof loader>>>() ?? []
+  const [items] = useState<ClientCredentialData[]>(client_credentials)
+  const defaultName = ''
+  const [name, setName] = useState(defaultName)
+  const defaultScope = 'gcn.nasa.gov/kafka-public-consumer'
+  const [scope, setScope] = useState(defaultScope)
+  const siteKey = 'site-key'
```

Comment:
Would you please restore the logic from main that gets this from the environment?

Reply:
Updated!